project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -3608,3 +3615,345 @@ function c4m_og_entity_delete($entity, $entity_type) {
     taxonomy_vocabulary_delete($tags->vid);
   }
 }
+
+/**
+ * Form builder.
+ *
+ * Merge group content.
+ */
+function c4m_og_merge_groups_build_form($form, $form_state, $group) {
+  $current_group = c4m_og_current_group();
+  $content_types = c4m_og_feature_enabled_content_types();
+  if (!empty($content_types)) {
+    $final_array = array_combine($content_types, $content_types);
+    $form['content_types'] = array(
+      '#title' => t('Select the different content types to merge to the target group. Note that all other  (unselected) content will be deleted permanently.'),
+      '#type' => 'checkboxes',
+      '#options' => $final_array,
+      '#default_value' => $final_array,
+    );
+  }
+
+  $form['users'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Merge users'),
+    '#default_value' => 1,
+  );
+
+  $form['target_group'] = array(
```

Comment:
Could you have a quick look at the styling please?
![screen shot 2018-02-06 at 10 23 49](https://user-images.githubusercontent.com/789034/35851298-ea584a74-0b27-11e8-80df-4a0d0240ed3f.png)